### PR TITLE
Fix ConnectOptions for twilio-video

### PR DESCRIPTION
**Pull request by ben8p**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://media.twiliocdn.com/sdk/js/video/releases/2.0.0-beta14/docs/global.html#connect__anchor
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/twilio-video/index.d.ts [hunk 1]
```diff
@@ -471,21 +473,43 @@ export type AudioTrackPublication = LocalAudioTrackPublication | RemoteAudioTrac
...
     iceServersTimeout?: number;
     iceTransportPolicy?: RTCIceTransportPolicy;
     insights?: boolean;
-    maxAudioBitRate?: number | null;
-    maxVideoBitRate?: number | null;
+    maxAudioBitrate?: number | null;
+    maxVideoBitrate?: number | null;
     name?: string | null;
     networkQuality?: boolean | NetworkQualityConfiguration;
+    region?: 'au1' | 'br1' | 'ie1' | 'de1' | 'jp1' | 'sg1' | 'us1' | 'us2' | 'gll' | 'us1-ix' | 'us2-ix' | 'ie1-ix' | 'de1-ix' | 'sg1-ix';
```
Comment (ktsn) [inline types/twilio-video/index.d.ts]: Where `'us1-ix' | 'us2-ix' | 'ie1-ix' | 'de1-ix' | 'sg1-ix'` are described? I couldn't find them in https://www.twilio.com/docs/video/ip-address-whitelisting#signaling-communication which is referred from the api reference.

Comment (ben8p) [reply types/twilio-video/index.d.ts]: Here:
https://www.twilio.com/blog/2017/12/introducing-gll-for-group-rooms.html

Comment (ben8p) [reply types/twilio-video/index.d.ts]: Found them here: https://www.twilio.com/docs/voice/voip-sdk/regions
But now I noticed it's `voice`. I made a mistake maybe...

Comment (ktsn) [reply types/twilio-video/index.d.ts]: I see. Can we exclude `*-ix` from the type as `ConnectOptions` docs only says [this table](https://www.twilio.com/docs/video/ip-address-whitelisting#signaling-communication) is the list of supported signaling regions?

Comment (ben8p) [reply types/twilio-video/index.d.ts]: Done

**Review by ktsn — APPROVED** [review]
LGTM

**Pull request merged**